src/api/app/components/workflow_artifacts_per_step_component.rb

```diff
@@ -34,6 +37,33 @@ def call
 
   private
 
+  def artifacts_for_set_flag(parsed_artifacts)
+    capture do
+      list_of_flags(parsed_artifacts[:flags])
+    end
+  end
+
+  def list_of_flags(flags)
+    flags.each do |flag|
+      concat(flag_step_sentence(flag))
+    end
+  end
+
+  def flag_step_sentence(flag)
+    path_details = package_or_project_path(flag)
+
+    tag.li do
+      concat('Set flag ')
+      concat(tag.b("#{flag[:type]} "))
+      concat("#{flag[:status]}")
```

Comment:
```suggestion
      concat("#{flag[:status]}d")
```

To have enable**d** and desable**d**